ui/packages/consul-ui/app/controllers/application.js

```diff
@@ -29,7 +24,7 @@ export default class ApplicationController extends Controller {
         // TODO: Currently we clear cache from the ember-data store
         // ideally this would be a static method of the abstract Repository class
         // once we move to proper classes for services take another look at this.
-        this.store.clear();
+        this.store.invalidate();
```

Comment:
We use the invalidate word in several other places in the codebase that all mean the same as what we are doing here - clearing the a cache. This is the only place we use it on the store so I decided to change it in this PR.